device-types/Cisco/ASR1001-X.yaml

```diff
@@ -0,0 +1,37 @@
+manufacturer: Cisco
+model: ASR1001-X
+slug: asr1001-x
+u_height: 1
+is_full_depth: true
+interfaces:
+  - name: GigabitEthernet0
+    type: 1000base-t
+    mgmt_only: true
+    name: GigabitEthernet0/0/0
+    type: 1000base-t
+    name: GigabitEthernet0/0/1
+    type: 1000base-t
+    name: GigabitEthernet0/0/2
+    type: 1000base-t
+    name: GigabitEthernet0/0/3
+    type: 1000base-t
+    name: GigabitEthernet0/0/4
+    type: 1000base-t
+    name: GigabitEthernet0/0/5
+    type: 1000base-t
+    name: TenGigabitEthernet0/0/0
+    type: 10gbase-x-sfpp
+    name: TenGigabitEthernet0/0/1
+    type: 10gbase-x-sfpp
+console-ports:
+  - name: con 0
+    type: rj-45
+power-ports:
+  - name: PSU0
+    type: iec-60320-c14
+    maximum_draw: 250
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 250
+device-bay:
```

Comment:
I'm not familiar with this device, but this appears to be a regular expansion slot (not modeled) and not a device bay.

Reply:
Maybe I don't understand what is meant by a device bay. I see a device bay as a slot in a network device that can take a module such as a WIC/HWIC/NM/NIM/SPA that provides additional interfaces (types and quantities). Is that incorrect usage?

Reply:
Every component is described in [the docs](https://netbox.readthedocs.io/en/stable/core-functionality/devices/#device-bays):

> Device bays represent the ability of a device to house child devices. For example, you might install four blade servers into a 2U chassis. The chassis would appear in the rack elevation as a 2U device with four device bays. Each server within it would be defined as a 0U device installed in one of the device bays. Child devices do not appear within rack elevations or the "Non-Racked Devices" list within the rack view.
>
> Child devices are first-class Devices in their own right: that is, fully independent managed entities which don't share any control plane with the parent. Just like normal devices, child devices have their own platform (OS), role, tags, and interfaces. You cannot create a LAG between interfaces in different child devices.
>
> Therefore, Device bays are not suitable for modeling chassis-based switches and routers. These should instead be modeled as a single Device, with the line cards as Inventory Items.